rancher/installing-rancher/installing-server/index.md

```diff
@@ -23,22 +23,22 @@ sudo docker run -d --restart=always -p 8080:8080 rancher/server
 
 #### Rancher UI
 
-The UI and API will be available on the exposed port `8080`. After the docker image is downloaded, it will take a minute or two before Rancher has successfully started. The IP of the machine will need to be public and accessible from the internet in order for Rancher to work.
+The UI and API will be available on the exposed port `8080`. After the docker image is downloaded, it will take a minute or two before Rancher has successfully started. 
 
-You can access the UI by going to the following URL: `http://<SERVER_IP>:8080`. The `<SERVER_IP` is the public IP address of the host that is running Rancher server.
+Navigate to the following URL: `http://<SERVER_IP>:8080`. The `<SERVER_IP` is the public IP address of the host that is running Rancher server.
 
-Once the UI is up and running, you can start [adding hosts]({{site.baseurl}}/rancher/rancher-ui/infrastructure/hosts/). After the hosts are setup, you can start adding [services]({{site.baseurl}}/rancher/rancher-ui/applications/stacks/adding-services/).
+Once the UI is up and running, you can start [adding hosts]({{site.baseurl}}/rancher/rancher-ui/infrastructure/hosts/). After the hosts are added into Rancher, you can start adding [services]({{site.baseurl}}/rancher/rancher-ui/applications/stacks/adding-services/) or launch templates from the [Rancher catalog]({{site.baseurl}}/rancher/rancher-ui/applications/catalog/).
 
 <a id="ldap"></a>
 
 ### Enabling Active Directory or OpenLDAP for TLS
 
-In order to enable Active Directory or OpenLDAP for Rancher server with TLS, the Rancher server container will need need to be started with the ldap certificate passed to the server. On your Linux machine with Docker installed, place the certificate in `/some/dir`. 
+In order to enable Active Directory or OpenLDAP for Rancher server with TLS, the Rancher server container will need to be started with the ldap certificate. On the Linux machine that you want to launch Rancher server on, save the certificate. 
 
-Start Rancher by bind mount the volume that has the certificate. 
+Start Rancher by bind mounting the volume that has the certificate. 
 
 ```bash
-sudo docker run -d --restart=always -p 8080:8080 -v /some/dir/cert.crt:/ca.crt rancher/server
+sudo docker run -d --restart=always -p 8080:8080 -v /dir_that_contains_the_cert/cert.crt:/ca.crt rancher/server
```

Comment:
lets call out explicitly inside the container it MUST be called /ca.crt 
its not immediately clear to folks.